lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -52,49 +53,111 @@
 }
 
 .nx-btn--primary:not([disabled]) {
-  &.nx-segmented-btn__main-btn {
-    border-right-color: var(--nx-swatch-blue-60);
+  &.nx-segmented-btn__main-btn, &.nx-segmented-btn__dropdown-btn {
+    border-color: var(--nx-color-primary-segmented-button-divider);
+    position: relative;
+
+    &::after {
```

Comment:
Is there a reason this isn't entirely defined in the focus block? It has no visible effect until the `border-color` is set, correct?

Reply:
> Is there a reason this isn't entirely defined in the focus block? It has no visible effect until the `border-color` is set, correct?

Yeah, I'm not sure why I put that there initially, it definitely makes more sense the way you've suggested: [83cd0fa](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/83cd0fa6313451b676280186f02bc3e0b3a2aa20). (Then realized I needed to be more specific for the border-radius definitions in the individual buttons, so moved those into a individual focus blocks here [2fa7ece](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/2fa7ecec9411e9260f7166e4ec0b0ccb2148f2f7))